docs/configuration.md

```diff
@@ -0,0 +1,87 @@
+<!--
+    Copyright 2019 VMware, Inc.
+    SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Precaution configuration
+
+Precaution is using a variety of tools under the hood without exposing
+implementation details and complexity of the underlying tools to the user.
+
+Concretely, this means that Precaution ignores any tool specific configuration
+files which may exist in the project's repository and performs scans based solely
+on Precaution specific configuration.
+
+Precaution provides you with the option to use optional `.precaution.yaml` file.
+This file **must** be in the root directory of your project.
+
+All paths in the options should be **absolute**.
+
+This file can be configured with the following options:
+
+| Option | Syntax |
+|---|---|
+| [Exclude](#exclude) | [Globs](#globs) |
+
+## Exclude
+
+This option can be used to tell Precaution to exclude files or folders from processing.
+
+You should use the [Globs](#globs) syntax when defining the exclude option.
```

Comment:
"should" implies you don't necessarily have to.  So is glob syntax required or not?

Reply:
You must but also if you don't use special Glob syntax symbols it will work too.
Like if you want to exclude a specific file you can with `src/main.go` but maybe I can't express this in this sentence.

Reply:
Perhaps something like:
```suggestion
This option accepts [Globs](#globs] syntax for defining patterns of files to exclude
```